openid-4-verifiable-presentations-1_0.md

```diff
@@ -791,6 +791,10 @@ This document also defines the following additional error codes and error descri
 
 - The value of the `request_uri_method` request parameter is neither `get` nor `post` (case-sensitive).
 
+`wallet_unavailable`:
+
+- The Wallet appears to be unavailable and therefore unable to respond to the request. This can be useful in situations where the user agent cannot invoke the Wallet and another component receives the request while the user still controls the user experience and wishes to continue the journey on the Verifier website. For example, this applies when using claimed HTTPS URIs handled by the Wallet provider in case the platform cannot or does not translate the URI into a platform intent to invoke the Wallet.
```

Comment:
I would use the term user-agent to clarify that's a specialized entity

Reply:
I'm not sure I'm following, could you expand on your suggestion please Guiseppe? Maybe suggest alternative text?